source/issuing-credentials-to-wallet.html.md.erb

```diff
@@ -97,22 +98,28 @@ This diagram shows the exchange of a credential between a government service and
 </html>
 </div>
 
-### [User authenticates with GOV.UK One Login to use your service](/issue-a-credential/authenticate-users/#authenticate-users-with-one-login)
+### User authenticates with GOV.UK One Login to use your service
+
+There is [more guidance on authenticating users with GOV.UK One Login](/issue-a-credential/authenticate-users/#authenticate-users-with-one-login)
 
 <ol>
   <li>Your user accesses your service.</li>
 
   <li>Your service authenticates the user with GOV.UK One Login.</li>
 
-  <li>Your service fetches the user’s <code>walletSubjectId</code> from the GOV.UK One Login <a href="https://docs.sign-in.service.gov.uk/integrate-with-integration-environment/authenticate-your-user/#retrieve-user-information"><code>/userinfo</code> API</a>.</li>
+  <li>Your service fetches the user’s <code>walletSubjectId</code> from the GOV.UK One Login <a href="https://docs.sign-in.service.gov.uk/integrate-with-integration-environment/authenticate-your-user/#retrieve-user-information"><code>/userinfo</code> endpoint</a>.</li>
 </ol>
 
-There is detailed guidance on [how GOV.UK One Login works](https://docs.sign-in.service.gov.uk/how-gov-uk-one-login-works/) in the GOV.UK One Login technical documentation.
+There's detailed guidance on [how GOV.UK One Login works](https://docs.sign-in.service.gov.uk/how-gov-uk-one-login-works/) in the GOV.UK One Login technical documentation.
+
+### Your service issues a credential offer
+
+There's more guidance on [generating and issuing a credential offer](/issue-a-credential/credential-offer/#credential-offer).
 
-### [Your service issues a credential offer](/issue-a-credential/credential-offer/#credential-offer)
+[//]: # (TODO:: look into making orderd lists contine in mark-down as per https://spec.commonmark.org/0.29/#lists example 272)
```

Comment:
```suggestion
```

I think comments have potential, but we need to discuss them and make a decision as a team before we add them to any live code. Maybe keep this in personal notes for now and then bring it to a team meeting.